AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -92,10 +92,10 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
 
     ngOnInit() {
         this.failedSubmission = false;
-        this.featureService
-            .getFeatureFlagByName('StaffMemberFeature')
-            .pipe(first())
-            .subscribe(result => (this.showStaffMemberFeature = result));
+        // this.featureService
+        //     .getFeatureFlagByName('StaffMemberFeature')
+        //     .pipe(first())
+        //     .subscribe(result => (this.showStaffMemberFeature = result));
```

Comment:
Is this shared between video and booking? I think it was mentioned but just want to clarify that we couldn't just disable this by setting the flag.

Reply:
Do you mean delete this code and set the showStaffMemberFeature flag to false?